### PR TITLE
taxonomy: add new 2022 infotri system

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: add new 2022 infotri system

image

### CODE REVIEW

File: taxonomies/packaging_recycling.txt [hunk 1]
```diff
@@ -32,14 +32,14 @@ pt:reutilizar, a reutilizar, para reutilizar, para ser reutilizado, a ser reutil
 en:recycle, to recycle, to be recycled, recyclable
 bg:за рециклиране, рециклирай, да се рециклира
 de:recyceln, zum Recyceln, recycelbar
-fr:recycler, à recycler, recyclable
+fr:bac de tri, recycler, à recycler, recyclable
```
Comment (stephanegigandet) [inline taxonomies/packaging_recycling.txt:35]: This should be a new child entry, "en:recycle in sorting bin, sorting bin"

File: taxonomies/packaging_recycling.txt [hunk 2]
```diff
@@ -32,14 +32,14 @@ pt:reutilizar, a reutilizar, para reutilizar, para ser reutilizado, a ser reutil
...
 hu:újrahasznosítható, újrahasznosítandó, újrahasznosításra
 pt:reciclável, a reciclar, para reciclar
 # associated_materials: en:glass, en:aluminium, en:steel
 
 <en:recycle
 en:recycle in glass bin, recycle in glass container, recycle with glass, in glass bin, in glass container
-fr:recycler dans le conteneur à verre, recycler dans le bac à verre, recycler avec le verre, dans le conteneur à verre, avec le verre
+fr:tri verre, bac verre, recycler dans le conteneur à verre, recycler dans le bac à verre, recycler avec le verre, dans le conteneur à verre, avec le verre
```
Comment (stephanegigandet) [inline taxonomies/packaging_recycling.txt:42]: This one is tricky. We probably should keep uniform names, starting with the action "recycler dans le conteneur à verre", the others can be added as synonyms. Then we can find ways to make suggestions work for synonyms, and maybe have short name properties if we need to display them in a specific way.

**Comment by benbenben2** [comment]
@teolemon, @stephanegigandet 

The suggestions from Stephane are applied in the last commit.
Please review the changes. Resolve conversations. And eventually approve and merge.

File: taxonomies/packaging_recycling.txt [hunk 3]
```diff
@@ -96,6 +96,11 @@ en:recycle as green waste, compost, green waste
 nl:recycle als GFT, gft
 fr:compostable, à composter
 
+#sorting bin
+<en:recylcle
```
Comment (stephanegigandet) [inline taxonomies/packaging_recycling.txt:100]:
```suggestion
<en:recycle
```

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**